docs/sources/setup/upgrade/_index.md

```diff
@@ -69,6 +69,21 @@ This new metric will provide a more clear signal that there is an issue with ing
 
 1. `frontend.embedded-cache.max-size-mb` Embedded results cache size now defaults to 100MB.
 
+#### Embedded cache metric changes
+
+- The following embedded cache metrics are removed. Instead use `loki_cache_fetched_keys`, `loki_cache_hits`, `loki_cache_request_duration_seconds` which instruments requests made to the configured cache (`embeddedcache`, `memcached` or `redis`).
+  - `querier_cache_added_total`
+  - `querier_cache_gets_total`
+  - `querier_cache_misses_total`
```

Comment:
If these are a one-to-one replacement, to avoid confusion I'd list them next to the parameter they're replacing.  (If they aren't a one-to-one replacement, then ignore this suggestion!)
```suggestion
- The following embedded cache metrics are removed. Instead use the new metrics, which instrument requests made to the configured cache (`embeddedcache`, `memcached` or `redis`).
  - `querier_cache_added_total` instead use `loki_cache_fetched_keys`
  - `querier_cache_gets_total` instead use `loki_cache_hits`
  - `querier_cache_misses_total` instead use `loki_cache_request_duration_seconds` 
```